packages/babel-preset-default/package.json

```diff
@@ -32,12 +32,12 @@
 	],
 	"main": "index.js",
 	"dependencies": {
-		"@babel/core": "^7.16.0",
-		"@babel/plugin-transform-react-jsx": "^7.16.0",
-		"@babel/plugin-transform-runtime": "^7.16.0",
-		"@babel/preset-env": "^7.16.0",
-		"@babel/preset-typescript": "^7.16.0",
-		"@babel/runtime": "^7.16.0",
+		"@babel/core": "7.25.7",
```

Comment:
Let's add the changelog entry for this package so it's clear that these packages were updated.

Reply:
Done in [`4c83b1c` (#65949)](https://github.com/WordPress/gutenberg/pull/65949/commits/4c83b1c9b00eaab51cb7c213de0aef87ac954a7e)